pkgs/top-level/all-packages.nix

```diff
@@ -8665,9 +8665,7 @@ with pkgs;
 
   neuron-notes = haskell.lib.compose.justStaticExecutables (haskell.lib.compose.generateOptparseApplicativeCompletion "neuron" haskellPackages.neuron);
 
-  ngrok = ngrok-2;
-
-  ngrok-2 = callPackage ../tools/networking/ngrok-2 { };
+  ngrok = callPackage ../tools/networking/ngrok-3 { };
```

Comment:
If we're dropping `-3` in the attributename, I'd say that `-3` should be dropped from the directory as well. Just to be consistent.

Reply:
You're right. Done.